schul-cloud/schulcloud-client
# ISC-129 windows tile support

This commit adds Windows Tile Support (not Live Tile for now).

I implemented it according to this tutorial: https://webdesign.tutsplus.com/tutorials/how-to-add-windows-tiles-to-your-website--cms-23099
and fixed some issues by reverse engineering the example page https://pokiapp.com/ from the tutorial
What I noticed: You can't test it without https. Windows won't use the referenced images. To test if the code works, I created a small demo on https://beta.hackedit.de with the same folder structure as used here.

Additionaly I cleaned up some unused files

# How to test
Open: https://beta.hackedit.de with Microsoft Edge
![image](https://user-images.githubusercontent.com/22987140/49516373-c692e300-f899-11e8-9df7-c8968c6c238e.png)
Check your Start-Menu

# Screenshots
<img width="585" alt="win-tile" src="https://user-images.githubusercontent.com/22987140/49516215-64d27900-f899-11e8-97d4-cb81692bddf8.PNG">
<img width="575" alt="win-tile-wide" src="https://user-images.githubusercontent.com/22987140/49516216-64d27900-f899-11e8-88f0-b20915fed6df.PNG">
